package.json

```diff
@@ -12,7 +12,8 @@
     "test:debug": "node --inspect-brk ./node_modules/.bin/jest --runInBand --watch",
     "start": "webpack-dev-server --open",
     "build": "babel src --out-dir lib",
-    "bundle": "cross-env NODE_ENV=production rollup -c rollup.config.js"
+    "bundle": "cross-env NODE_ENV=production rollup -c rollup.config.js",
+    "lint": "eslint ./ --fix"
```

Comment:
apply eslint only on the source file. Otherwise, it will try to apply eslint fix on the distribution file whic is not expected.

Reply:
fort that we need to create a eslintignore file, i will create that